src/Transition/CreateNewIDFUsingRulesV9_6_0.f90

```diff
@@ -426,7 +426,16 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
               ! If your original object starts with R, insert the rules here
 
               ! If your original object starts with S, insert the rules here
-
+			  CASE('PERFORMANCEPRECISIONTRADEOFFS')
```

Comment:
Mixed tab/space here causing weird indent, I'll fix it since I need to resolve a tiny conflict anyway.

Reply:
@Myoldmopar thank you!